### PR TITLE
add freemind

### PR DESCRIPTION
**Pull request by raisercostin**

### CODE REVIEW

File: freemind.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "http://freemind.sourceforge.net/wiki/index.php/Main_Page",
+    "version": "1.0.1",
+    "description": "FreeMind is a free mind mapping application written in Java. It provides extensive export capabilities.",
+    "license": "GNU GPL V2+",
+    "hash": "005e815bf5d1867eadcaa89194df9952d986ace9eed0a3058c7ee6255799a60a",
+    "url": "https://netix.dl.sourceforge.net/project/freemind/freemind/1.0.1/freemind-bin-max-1.0.1.zip",
```
Comment (Ash258) [inline freemind.json]: Use sourceforges direct download mirror: `https://downloads.sourceforge.net/project/freemind/freemind/1.0.1/freemind-bin-max-1.0.1.zip` with hash `8b4c16f253d5c56b7fddf6a1878831556febfc48` found in https://sourceforge.net/projects/freemind/files/freemind/1.0.1/

Comment (brandon93s) [reply freemind.json]: Oops, merged.

**Comment by brandon93s** [comment]
Autoupdate added: https://github.com/lukesampson/scoop-extras/commit/4d60e2135d6c35b312537f814f155358ddf891c8

**Comment by Ash258** [comment]
It's pretty much irrelevant since last version was published in 2014 😂 But for that hash calculation it's good.

**Pull request merged**